data/human/human missions.txt

```diff
@@ -6120,3 +6120,43 @@ mission "Blind Man from Martini"
 				clear "Blind Man from Martini: label far"
 				clear "Blind Man from Martini: label looking"
 			`	With that, he grabs his cane and his luggage, and follows a hospital attendant toward his daughter's room, leaving you in the reception.`
+
+mission "Republic Navy Advisory System [frozen]"
+	name "Cold Relief"
+	description "Humanitarian aid of <cargo> to <destination> following a severe cold snap."
+	minor
+	repeat
+	source
+		government "Republic"
+	destination
+		goverment "Republic" "Syndicate" "Independent" "Neutral"
```

Comment:
```suggestion
		government "Republic" "Syndicate" "Independent" "Neutral"
```

Reply:
I think this has been fixed already—https://github.com/endless-sky/endless-sky/pull/8871/commits/e0787b2f16ec3e7936cc46f37974ffbeeee8c95e